composer.json

```diff
@@ -74,9 +75,15 @@
                 "web-root": "./build"
             }
         },
+        "drush": {
+            "services": {
+                "drush.services.yml": "^11"
+            }
+        },
```

Comment:
This won't be needed, please see my issue comment [here](https://github.com/openeuropa/oe_paragraphs/issues/217#issuecomment-1907854259).